onnx_tf/handlers/backend/resize.py

```diff
@@ -18,6 +37,74 @@ def args_check(cls, node, **kwargs):
+      # | cubic   | align_corners                  | -0.5          | 1               | not apply           | not apply         | supported (1) | supported |
+      # |---------|--------------------------------|---------------|-----------------|---------------------|-------------------|---------------|-----------|
+      # | cubic   | asymmetric                     | -0.5          | 1               | not apply           | not apply         | supported (1) | supported |
+      # |---------|--------------------------------|---------------|-----------------|---------------------|-------------------|---------------|-----------|
+      # | nearest | tf_crop_and_resize             | not apply     | 0               | any float value     | round_prefer_ceil | supported     | supported |
+      # |---------|--------------------------------|---------------|-----------------|---------------------|-------------------|---------------|-----------|
+      # | linear  | tf_crop_and_resize             | not apply     | 0               | any float value     | not apply         | supported     | supported |
+      # |---------|--------------------------------|---------------|-----------------|---------------------|-------------------|---------------|-----------|
+      # Note:
+      # 1. The accuracy of your model will go down, if the height and the width of the new sizes(scales * origial sizes) are not in whole numbers.
+      coordinate_transformation_mode = node.attrs.get(
+          "coordinate_transformation_mode", "half_pixel")
+      cubic_coeff_a = node.attrs.get("cubic_coeff_a", -0.75)
+      exclude_outside = node.attrs.get("exclude_outside", 0)
+      mode = node.attrs.get("mode", "nearest")
+      nearest_mode = node.attrs.get("nearest_mode", "round_prefer_floor")
+      if coordinate_transformation_mode == "pytorch_half_pixel":
+        exception.OP_UNSUPPORTED_EXCEPT(
+            "Resize coordinate_transformation_mode=pytorch_half_pixel",
+            "Tensorflow")
+      if (coordinate_transformation_mode == "half_pixel" and mode == "nearest"
+         ) or (coordinate_transformation_mode == "tf_half_pixel_for_nn" and
+               mode in ["linear", "cubic"]) or (
+                   coordinate_transformation_mode == "tf_crop_and_resize" and
+                   mode == "cubic"):
+        exception.OP_UNSUPPORTED_EXCEPT(
+            "Resize coordinate_transformation_mode=" +
+            coordinate_transformation_mode + " and  mode=" + mode, "Tensorflow")
+      if (exclude_outside == 1 and
+          mode in ["nearest", "linear"]) or (exclude_outside == 0 and
+                                             mode == "cubic"):
+        exception.OP_UNSUPPORTED_EXCEPT(
+            "Resize mode=" + mode + " and exclude_outside=" +
+            str(exclude_outside), "Tensorflow")
+      if cubic_coeff_a == -0.75 and mode == "cubic":
```

Comment:
Based on the support table, I think this should be checking
```if cubic_coeff_a != -0.5 and mode == "cubic":```

Reply:
@pluradj you are right -0.5 and -.75 are the common one but not the only values. I have updated the code, please review it again. Thanks